frontend/src/js/components/settings/organization/Billing.tsx

```diff
@@ -172,7 +213,46 @@ export const Billing = () => {
             </div>
           }
         />
-        {!organization.trial && !isEnterprise && <OrganizationPaymentSettings />}
+        {billing && changeBilling && (
+          <PlanExpanded
+            isEdit
+            onCloseClick={() => setChangeBilling(false)}
+            currentBillingProfile={{ address: { ...billing.address }, name: billing.name, email: billing.email }}
+            card={card}
+          />
+        )}
+        <div className={classes.billingSection}>
+          <div className="flexbox center-aligned">
+            <div className={classes.emailTooltip}>
+              <b>Billing details</b>
+            </div>
+            {!isEnterprise && billing && billing.address && (
+              <Button className="margin-left" onClick={() => setChangeBilling(true)}>
+                Edit
+              </Button>
+            )}
+          </div>
+          {isEnterprise ? (
+            <InfoText>
+              Enterprise plan payments are invoiced periodically to your organization. If you have any questions about your billing, <br /> please contact{' '}
+              <a href="mailto:support@mender.io" target="_blank" rel="noopener noreferrer">
+                support@mender.io
+              </a>
+            </InfoText>
+          ) : billing && billing.address ? (
+            <div className="flexbox">
+              <Address address={billing.address} email={billing.email} name={billing.name} />
+              {card && <CardDetails card={card} containerClass={classes.cardDetails} />}
```

Comment:
Sidenote: you could keep these two both free from a passed class with slightly different css.